ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -324,6 +334,39 @@ StackLayout {
             }
         }
     }
+
+    Component {
+        id: controlNodeOffline
+        ControlNodeOfflineCommunityView {
+            id: joinCommunityView
```

Comment:
copy paste error with the id?

Reply:
yes, I it seems I didn't commit this changes